app/assets/scripts/vendor/map.js

```diff
@@ -2,14 +2,18 @@
       // parameter when you first load the API. For example:
       // <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places">
 
+      var map;
+      var markers = [];
+
       function initMap() {
-        var map = new google.maps.Map(document.getElementById('map'), {
+        map = new google.maps.Map(document.getElementById('map'), {
           center: {lat: -33.8688, lng: 151.2195},
```

Comment:
It looks like the JS in our existing JS files uses 4-space tabs instead of 2-space tabs. Can you (either in this PR or future PR) update the spacing in this file? (Unless github is lying and this file actually uses 4-space tabs...)